include/aspect/material_model/visco_plastic.h

```diff
@@ -156,6 +156,16 @@ namespace aspect
 
         double get_min_strain_rate() const;
 
+        /**
+         * A function that computes the plastic yielding
+         * if needed outside material model (e.g., particles).
```

Comment:
I think the name of the function currently does not make clear what `plastic yielding` means, how do you return it as a bool? What about the following function name: `is_yielding`? You could then update the documentation to:
```
A function that returns whether the material is plastically yielding at the given pressure, temperature, composition, and strain rate.
```